src/WooCommerceAjax.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * WooCommerce Ajax related functionality.
+ */
+class WooCommerceAjax {
+
+    /**
+     * WooCommerce Ajax initialization method.
+     */
+    public static function init() 
+    {
+
+
+        add_action('wp_ajax_woocommerceAjaxAddToCart', __CLASS__ . '::woocommerceAjaxAddToCart');
+        add_action('wp_ajax_nopriv_woocommerceAjaxAddToCart', __CLASS__ . '::woocommerceAjaxAddToCart');
+        add_filter('woocommerce_add_to_cart_fragments', __CLASS__ . '::ajaxAddToCartNoticeFragments');
+        // add_filter('woocommerce_add_to_cart_fragments', __NAMESPACE__ . '\WooCommerceAjax::ajaxAddToCartAccessoriesFragments');
+    }
+
+    /**
+     * WooCommerce Ajax initialization method.
+     * 
+     * @return object
+     */
+    public static function woocommerceAjaxAddToCart() 
+    {
+        $product_id = apply_filters('woocommerce_add_to_cart_product_id', absint($_POST['product_id']));
+        $quantity = empty($_POST['quantity']) ? 1 : wc_stock_amount($_POST['quantity']);
+        $variation_id = absint($_POST['variation_id']);
+        $passed_validation = apply_filters('woocommerce_add_to_cart_validation', true, $product_id, $quantity);
+        $product_status = get_post_status($product_id);
+
+        if ($passed_validation && WC()->cart->add_to_cart($product_id, $quantity, $variation_id) && 'publish' === $product_status) {
+            do_action('woocommerce_ajax_added_to_cart', $product_id);
+            wc_add_to_cart_message(array($product_id => $quantity), true);

```

Comment:
```suggestion
            wc_add_to_cart_message(array($product_id => $quantity), TRUE);
```
Are you using our coding standards ruleset? There seems to be a a few code format problems with this PR.